conode/run_conode.sh

```diff
@@ -0,0 +1,345 @@
+#!/usr/bin/env bash
```

Comment:
You'd have to be able to import files after the compilation - I don't know how. This is one reason I put most of the functionality in `onet/app`, so the `conode.go` is very lightweight. I corrected the current conode.go and really do use the `onet/app`, which was not the case before.

Thanks for keeping track of my changes ;)